core/jvm/src/main/scala/bobcats/KeyPlatform.scala

```diff
@@ -16,19 +16,17 @@
 
 package bobcats
 
-import java.security
 import javax.crypto
 
-private[bobcats] trait KeyPlatform {
-  def toJava: security.Key
-}
```

Comment:
Why did this change? A `KeySpec` is different from a `Key`, and I think it makes sense to have `Key` here. That way we don't need to decide which `KeySpec` to use.

Reply:
I was having difficulty compiling things, and I thought the reason was a clash here. I'll look at it more carefully again, and see what the problem was.

Reply:
ok I fixed this. The `toJava` is back.